data/raw/iam/readme.md

```diff
@@ -21,10 +21,24 @@ ls -1 *.png | parallel --eta -j 6 convert '{}' -adaptive-resize 50% '../forms-re
 
 ## Split
 
-The data split we will use is
-IAM lines Large Writer Independent Text Line Recognition Task (lwitlrt): 9,862 text lines.
-
-- The validation set has been merged into the train set.
-- The train set has 7,101 lines from 326 writers.
-- The test set has 1,861 lines from 128 writers.
-- The text lines of all data sets are mutually exclusive, thus each writer has contributed to one set only.
+The data split we will use is loosely based on the IAM lines Large Writer Independent Text Line Recognition Task (`lwitlrt`) which provides 4 data splits:
```

Comment:
do you have a link to the LWITRT? would be good to share it here to add context, and a google search doesn't immediately turn up anything.

even though i've worked with this data a bit, i don't have sufficient context to really appreciate what's being discussed here

also, smaller corrections: capital L in Lines, "is loosely based"

Reply:
all I can find is https://fki.tic.heia-fr.ch/databases/iam-handwriting-database which is already present in the readme! 
I fixed the capital "L".